khmer/khmer_args.py

```diff
@@ -110,6 +263,15 @@ def __call__(self, parser, namespace, values, option_string=None):
 ** Your values for ksize, n_tables, and tablesize
 ** will be ignored.'''.format(hashfile=values))
 
+            if getattr(namespace, 'unique_kmers') != 0 or \
+               getattr(namespace, 'max_memory_usage') is not None or \
```

Comment:
you can remove 'is not None'. 0 (which is the alternative 'false' here) is not a sane argument either.